challenges/how-to-create-a-list.py

```diff
@@ -0,0 +1,31 @@
+# The challenge is to write a script that can generate all the fibonacci numbers form 1 to 100
+
+def using_normal_loops():
+    fibonacci = [1, 1]
+    for i in range(2, 100):
+        previous_number_1 = fibonacci[i - 1]
+        previous_number_2 = fibonacci[i - 2]
+        new_number = previous_number_1 + previous_number_2
+        fibonacci.append(new_number)
+
+    for idx, val in enumerate(fibonacci):
+        print(f"{idx+1}: {val}")
+
+def using_recursion(n):
+    # Point to be noted: Generating fibonacci numbers with recursion is really slow
+    # To speed things up, concepts like memoizition or dynamic programming is applicable
+    if n == 0:  return 1
+    elif n == 1: return 1
+    else: return (using_recursion(n - 1) + using_recursion(n - 2))
+
+using_normal_loops()
+for i in range(101):
+    print(f"{i+1}: {using_recursion(i)}")
+
+# Using list comprehension
+    # This is nothing different than the first one. But in a shorter manner
+
+fibonacci = [1, 1]
+[fibonacci.append(fibonacci[i - 1] + fibonacci[i - 2]) for i in range(2, 100)]
```

Comment:
Interesting solution! I generally don't love list comprehensions being used this way. After all, it's being used to update an external list, but a list is still generated (filled with None). That said, it gets the job done! 👍

Reply:
I also don't like list comprehensions like this. But just to prove, "If you try, you can solve any problem."
😉

Reply:
Hahah yeah, fair enough! I'm happy with it if you are.